control/controlclient/netmap.go

```diff
@@ -271,10 +271,6 @@ func (nm *NetworkMap) WGCfg(logf logger.Logf, flags WGConfigFlags) (*wgcfg.Confi
 		if Debug.OnlyDisco && peer.DiscoKey.IsZero() {
 			continue
 		}
-		if (flags&AllowSingleHosts) == 0 && len(peer.AllowedIPs) < 2 {
```

Comment:
Fixes https://github.com/tailscale/tailscale/issues/1224 in commit message?

Reply:
Done.